torch/_dynamo/variables/tensor.py

```diff
@@ -903,6 +903,11 @@ def call_method(
         from torch._dynamo.variables.builder import wrap_fx_proxy_cls
         from ..utils import numpy_method_wrapper
 
+        if name == "__len__":
+            # `TensorVariable` knows how to handle this.
+            # Also, we don't need `numpy_method_wrapper` as `len` returns an `int`.
+            return super().call_method(tx, "size", [ConstantVariable(0, **options)], {})
```

Comment:
There are a number of methods of this sort in https://github.com/Quansight-Labs/numpy_pytorch_interop/blob/main/torch_np/_ndarray.py. I would like if it'd be possible to handle all of them
1. All via the same logic
2. Without having to reimplement the logic, simply tracing the implementation in `torch_np._ndarray`